WasteWar/Assets/Scripts/Camera Controls/Zoom.cs

```diff
@@ -0,0 +1,96 @@
+﻿using UnityEngine;
+
+public class Zoom : MonoBehaviour
+{
+    [SerializeField]
+    private Transform route;
+
+    [SerializeField]
+    private Transform cam;
+
+    [SerializeField]
+    private Transform rotationRoute;
+
+    //parameter for parametric equation
+    public float t = 0;
+
+    //points along Bezier curve for smooth movement of camera
+    private Vector3 p1;
+
+    private Vector3 p2;
+
+    private Vector3 p3;
+
+    private Vector3 p4;
+   
+    void Start()
+    {
+
+        p1 = route.GetChild(0).position;
+        p2 = route.GetChild(1).position;
+        p3 = route.GetChild(2).position;
+        p4 = route.GetChild(3).position;
+
+        while (transform.rotation.eulerAngles.x > 60)
+        {
+            t += Time.deltaTime * CameraConstants.ZOOM_SPEED;
+            cam.rotation = Quaternion.Lerp(
+                    rotationRoute.GetChild(0).rotation,
+                    rotationRoute.GetChild(1).rotation,
+                    t);
+            cam.position = Mathf.Pow(1 - t, 3) * p1 +
```

Comment:
don't use actual numbers, pull them into consts..

Reply:
cam.position = CalculatePosition()

Reply:
these are not actual numbers, it's an equation